polishNotationCalc.cpp

```diff
@@ -0,0 +1,92 @@
+#include "polishNotationCalc.hpp"
+namespace dev
+{
+    PolishCalc::PolishCalc()
+    {
+        insertFunctionInsideMap();
+    }
+
+    bool PolishCalc::isNumeric(const char num)
+    {
+        double temp = num - '0';
+        if (temp >= 0 && temp <= 9)
+            return true;
+
+        return false;
+    }
+    bool PolishCalc::isOperator(const char num)
+    {
+        if (num == '+' || num == '-' || num == '*' || num == '/')
+            return true;
+        return false;
+    }
+   
+    double PolishCalc::calc( const std::string &str)
```

Comment:
3 blocks : 
```
1. Basic Validation // Check that the is not bad symbols and so on
2. Parsing // Use not strings, but OOP types or lambdas for futher working 
2.1 Validation : count of operator matches count of operands 
3. Calculation // Evaluete expression 
```


2 stacks : 
Validation block fill up 2 stacks : 
 1. Operator stack
 2. Operand stack

Calculation should work with Operator/Operand or lambda types but not string.